e2e_samples/parking_sensors/devops/azure-pipelines-ci-artifacts.yml

```diff
@@ -62,12 +64,32 @@ stages:
         ArtifactName: 'databricks'
       displayName: 'Publish Databricks Artifacts'
 
+    - task: Npm@1
+      inputs:
+        command: 'install'
+        workingDir: '$(Build.Repository.LocalPath)/$(basePath)/adf' #replace with the package.json folder
+        verbose: true
+      displayName: 'Install npm package'
+
+    - task: Npm@1
+      inputs:
+        command: 'custom'
+        workingDir: '$(Build.Repository.LocalPath)/$(basePath)/adf' #replace with the package.json folder
+        customCommand: 'run build export $(Build.Repository.LocalPath)/$(basePath)/adf $(adfResourceId) "arm-template"'
+      displayName: 'Validate and Generate ARM template'
```

Comment:
Please add a comment referring to this two links in the code and in the ReadMe as well: 
- https://learn.microsoft.com/en-us/answers/questions/282058/automated-publishing-of-data-factory-with-devops
- https://microsoft-bitools.blogspot.com/2021/09/adf-deploy-missing-publishconfigjson.html